BricklinkSharp.Client/UpdateInventory.cs

```diff
@@ -40,7 +40,7 @@ public class UpdateInventory
     public decimal? UnitPrice { get; set; }
 
     [JsonPropertyName("description")]
-    public string? Description { get; set; }
+    public string Description { get; set; } = String.Empty;
```

Comment:
Is there a reason for this change?

Reply:
yes, indeed. A null value is not accepted by Bricklink API. I don't if this was a change or if it was always the case.

Reply:
I See. But sending an empty string will overwrite an existing description without the User will notice (breaking), or in other words: one must send the description to not delete it.

 I remember that I made this field nullable in purpose to avoid this overwrite behavior. My guess is, that Not the BL API changed but the way the .net json serializer treats null. Instead of the old behavior (do Not append a null property) it now adds a description property with null value and that new property is refused by BL.

There is an Option for the Json serializer to Not add null properties. Can you verify the outgoing requests Body?

Br Jens

Reply:
For whatever reason it works again with a null description property.